mediacapture-image/getPhotoSettings.html

```diff
@@ -43,4 +43,19 @@
 
 }, 'getPhotoSettings() of an ended Track should throw "InvalidStateError"');
 
+async_test(t => {
+  let canvas = document.getElementById('canvas');
+  let context = canvas.getContext('2d');
+  context.fillStyle = 'red';
+  context.fillRect(0, 0, 10, 10);
+  let stream = canvas.captureStream();
+  let videoTrack = stream.getVideoTracks()[0];
+
+  let capturer = new ImageCapture(videoTrack);
+  capturer.getPhotoSettings()
+  .then(t.step_func_done(() => assert_unreached('should throw "OperationError"')))
+  .catch(t.step_func_done(e => assert_equals(e.name, 'OperationError')))
+  videoTrack.stop();
+}, 'throw "OperationError" when the MediaStreamTrack being ended asynchronously');
```

Comment:
'throw "OperationError" when the MediaStreamTrack is stopped while getting photo settings'